target/linux/ath79/dts/qca9533_tplink_tl-wr802n-v1.dts

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "qca9533_tplink_tl-wr802n.dtsi"
+
+/ {
+	model = "TP-Link TL-WR802N v1";
+	compatible = "tplink,tl-wr802n-v1", "qca,qca9533";
+
+	keys {
+		reset {
+			gpios = <&gpio 11 GPIO_ACTIVE_HIGH>;
+		};
+	};
+};
+
+&led_system {
+			label = "tp-link:blue:system";
```

Comment:
On the data sheet it looks green as well ...

Reply:
V1 manual I got from fccid.io also shows green LED, so I'll stick with it for both versions. It's possible that it was changed from blue to green in minor revision of V1.

Reply:
Unified both of them